cmd/lotus/daemon.go

```diff
@@ -173,16 +179,21 @@ var DaemonCmd = &cli.Command{
 			return xerrors.Errorf("initializing node: %w", err)
 		}
 
-		// Add lotus version info to prometheus metrics
-		var lotusInfoMetric = promauto.NewGaugeVec(prometheus.GaugeOpts{
-			Name: "lotus_info",
-			Help: "Lotus version information.",
-		}, []string{"version"})
-
-		// Setting to 1 lets us multiply it with other stats to add the version labels
-		lotusInfoMetric.With(prometheus.Labels{
-			"version": build.UserVersion,
-		}).Set(1)
+		// We are using this metric to tag info about lotus even though
+		// it doesn't contain any actual metrics
+		if err = view.Register(
+			&view.View{
+				Name:        "info",
+				Description: "Lotus node information",
+				Measure:     lotusInfo,
+				Aggregation: view.LastValue(),
+				TagKeys:     []tag.Key{version},
+			},
+		); err != nil {
+			log.Fatalf("Cannot register the view: %v", err)
+		}
+		// Set the metric to one so it is published to the exporter
+		stats.Record(ctx, lotusInfo.M(1))
```

Comment:
We'll eventually move this into the DI container, but it's good enough here for now